cypress/support/commands.ts

```diff
@@ -42,10 +42,8 @@ export function assertNoTab(tabName: string): any {
 export function resetDB() {
   before(() => {
     cy.exec('java -jar dockstore-webservice.jar db drop-all --confirm-delete-everything travisci/web.yml');
-    cy.exec('PGPASSWORD=dockstore psql -h localhost -f travisci/db_dump.sql webservice_test -U dockstore')
-      .its('stdout').should('contain', 'ALTER TABLE');
-    cy.exec('java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0 travisci/web.yml')
-      .its('stdout').should('contain', 'Successfully released change log lock');
+    cy.exec('PGPASSWORD=dockstore psql -h localhost -f travisci/db_dump.sql webservice_test -U dockstore');
```

Comment:
This seems to be removing tests, intentional?

Reply:
Originally I was using it to debug the reset postgres timeout issue.  Didn't seem to give me any useful information.  Removing it because it's too noisy.